sonar-plugin-api/src/main/java/org/sonar/api/database/DatabaseProperties.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.sonar.api.database;
 
+/**
+ * @deprecated since 7.1, plugins don't connect to database
+ */
+@Deprecated
 public interface DatabaseProperties {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-10300%2Fforbid_system_properties_in_settings_ws&issues=AWEnWsQUgvt_XO9IEnRk&open=AWEnWsQUgvt_XO9IEnRk)

[ISSUE_KEY]: AWEnWsQUgvt_XO9IEnRk